samples/bluetooth/beacon/sample.yaml

```diff
@@ -4,5 +4,5 @@ sample:
 tests:
   test:
     harness: bluetooth
-    platform_whitelist: qemu_cortex_m3 qemu_x86 nrf52_pca10040
```

Comment:
concerned about these changes to the YAML files.
shouldn't the new name be on these whitelists?
else the test just won't be run under emulation unless I'm missing something

Reply:
it will run under qemu_x86